apps/flow-docs/app/routes/action/set-theme.tsx

```diff
@@ -0,0 +1,26 @@
+import type { ActionFunction, LoaderFunction } from "@remix-run/node";
+import { json, redirect } from "@remix-run/node";
+import { getThemeSession } from "~/utils/Theme.server";
+import { isTheme } from "~/utils/ThemeProvider";
+
+export const action: ActionFunction = async ({ request }) => {
+  const themeSession = await getThemeSession(request);
+  const requestText = await request.text();
+  const form = new URLSearchParams(requestText);
+  const theme = form.get("theme");
+
+  if (!isTheme(theme)) {
+    return json({
+      success: false,
+      message: `theme value of ${theme} is not a valid theme`,
+    });
```

Comment:
Do we need to return a default theme here? Or, is it handled elsewhere?

Reply:
Yes, the frontend defaults to a light theme